packages/react/src/components/MapCard/storyFiles/MapboxExample.jsx

```diff
@@ -0,0 +1,315 @@
+  options,
+  isLegendFullWidth,
+  onCardAction,
+  availableActions,
+  isSettingPanelOpen,
+  isExpanded,
+  isResizable,
+  ...other
+}) => {
+  const mapContainerRef = useRef(null);
+  const [active, setActive] = useState(options[0]);
+  const [map, setMap] = useState(null);
+  const [activeSideBar, setActiveSideBar] = useState(1);
+  const [selectedControls, setSelectedControls] = useState([]);
+
+  const changeState = (i) => {
+    setActive(options[i]);
+  };
+
+  const toggleControlSelection = (id) => {
+    setSelectedControls((state) =>
+      selectedControls.includes(id) ? state.filter((currentId) => currentId !== id) : [...state, id]
+    );
+  };
+
+  const mapControls = [
+    {
+      hasScroll: true,
+      visibleItemsCount: 4,
+      group: [
+        {
+          icon: Hail32,
+          iconDescription: 'Map scroll hail',
+          onClick: () => changeState(0),
+        },
+        {
+          icon: Rain32,
+          iconDescription: 'Map scroll rain',
+          onClick: () => changeState(0),
+        },
+        {
+          icon: Fog32,
+          iconDescription: 'Map scroll fog',
+          onClick: () => changeState(0),
+        },
+        {
+          icon: Cloudy32,
+          iconDescription: 'Map scroll cloudy',
+          onClick: () => changeState(1),
+        },
+        {
+          icon: Cloud32,
+          iconDescription: 'Map scroll cloud',
+          onClick: () => changeState(0),
+        },
+        {
+          icon: PartlyCloudy32,
+          iconDescription: 'Map scroll partly cloudy',
+          onClick: () => changeState(0),
+        },
+
+        {
+          icon: Sun32,
+          iconDescription: 'Map scroll sun',
+          onClick: () => changeState(0),
+        },
+      ],
+    },
+    {
+      group: [
+        {
+          icon: Events32,
+          iconDescription: 'GDP',
+          onClick: () => changeState(1),
+        },
+        {
+          icon: Events32,
+          iconDescription: 'Population',
+          onClick: () => changeState(0),
+        },
+        {
+          icon: Events32,
+          iconDescription: 'Map control 3A',
+          onClick: () => changeState(1),
+        },
+      ],
+    },
+    {
+      id: 'toggle1',
+      kind: 'icon-selection',
+      selected: selectedControls.includes('toggle1'),
+      icon: Events32,
+      iconDescription: 'Toggle control 1',
+      onClick: () => toggleControlSelection('toggle1'),
+    },
+    {
+      id: 'toggle2',
+      kind: 'icon-selection',
+      selected: selectedControls.includes('toggle2'),
+      icon: Events32,
+      iconDescription: 'Toggle control 2',
+      onClick: () => toggleControlSelection('toggle2'),
+    },
+    {
+      id: 'toggle3',
+      kind: 'icon-selection',
+      selected: selectedControls.includes('toggle3'),
+      icon: Events32,
+      iconDescription: 'Toggle control 3',
+      onClick: () => toggleControlSelection('toggle3'),
+    },
+  ];
+
+  const layeredControls = [
+    {
```

Comment:
Is there an option to swap the tooltip direction and alignment for all custom buttons? I can see it being useful to move for situations like this:

![Screen Shot 2021-06-01 at 8 17 47 AM](https://user-images.githubusercontent.com/25177649/120329935-f2984f00-c2b1-11eb-885f-35e599de6590.png)

Reply:
Nope, that is currently not possible. I don't know if the alternatives are better in this case:
![image](https://user-images.githubusercontent.com/5167091/120531147-9aab3680-c3de-11eb-837b-d7cc22050d2c.png)

Reply:
@davidicus Do you think we need to open a new issue for this? It is pretty annoying but I don't see a quick fix.

Reply:
I think tooltipDirection: bottom, and tooltipAlignment: left could work, but not a deal breaker